src/SignalR/common/Protocols.Json/src/Protocol/JsonHubProtocol.cs

```diff
@@ -0,0 +1,777 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Runtime.ExceptionServices;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Internal;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.SignalR.Protocol
+{
+    /// <summary>
+    /// Implements the SignalR Hub Protocol using System.Text.Json.
+    /// </summary>
+    public class JsonHubProtocol : IHubProtocol
```

Comment:
Should this be sealed?

Reply:
I don't see a reason to allow or disallow inheriting this class.

Reply:
Let's use sealed by default for new public classes.